cadcutils/cadcutils/net/ws.py

```diff
@@ -455,84 +455,40 @@ def upload_file(self, url, src, md5_checksum=None, **kwargs):
            :throws: HttpExceptions
         """
         stat_info = os.stat(src)
-        with_transactions = False
+        if stat_info.st_size == 0:
```

Comment:
This is a pretty big change in behaviour (I know this because there were about 7000 empty files in the SUBARUCADC collection).

Do you really want to dis-allow sentinel files in vos?

Reply:
Empty files are not allowed in SI. In vos, empty files are stored as DataNode without pointer to the data (just metadata in DB).
What are the SUBARUCADC empty files? You should flag it out in slack if they need to be stored in SI so that alternatives can be considered.